### DISCUSSION TITLE
Writing Gleam Tutorial Question: why I have to set `format_pair` as ` pub fn` to make it work?

### DISCUSSION THREAD

**Discussion by shenxiangzhuang** [gleam-lang/gleam#2641] [Questions & support] [answered]
I'm fellowing the tutorial in [Writing Gleam Learn to work with Gleam projects](https://gleam.run/writing-gleam/), everything goes well except that I have to define `format_pair` with `pub fn` rather tha `fn` as the docs: 
```gleam
pub fn format_pair(name: String, value: String) -> String {
  name <> "=" <> value
}
```
If I define `format_pair` with `fn` , I got error:
```
➜  vars git:(main) ✗ gleam run -m gleescript
  Compiling vars

warning: Unused private function
  ┌─ /Users/mathew/personal/learn/gleam/vars/src/vars/internal.gleam:1:1
  │
1 │ fn format_pair(name: String, value: String) -> String {
  │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ This private function is never used

Hint: You can safely remove it.
error: Unknown module field
   ┌─ /Users/mathew/personal/learn/gleam/vars/src/vars.gleam:18:22
   │
18 │   io.println(internal.format_pair(name, value))
   │                      ^^^^^^^^^^^^

The module `vars/internal` does not have a `format_pair` value.
```

## Screenshot
![image](https://github.com/gleam-lang/gleam/assets/17157965/2185815d-2af6-4839-bd10-d208e9e76d8f)

Maybe I made some trivial mistakes, any help would be greatly appreciated!

**Comment by lpil** [comment, marked as answer]
Hello! That was a typo in the documentation, sorry about that. I have now fixed it.
